application/packages/ContactUs/src/Service/ContactUsService.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace ContactUs\Service;
+
+use ContactUs\Filter\ContactUsFilter;
+use ContactUs\Mapper\ContactUsMapper;
+use MysqlUuid\Formats\Binary;
+use Ramsey\Uuid\Uuid;
+use Std\FilterException;
+use Zend\Paginator\Paginator;
+
+/**
+ * Class ContactUsService
+ *
+ * @package ContactUs\Service
+ * @author  Djordje Stojiljkovic <djordjestojilljkovic@gmail.com>
+ */
+class ContactUsService
+{
+    /**
+     * @var ContactUsFilter $contactUsFilter
+     */
+    private $contactUsFilter;
+
+    /**
+     * @var ContactUsMapper $contactUsMapper
+     */
+    private $contactUsMapper;
+
+    /**
+     * @var Paginator $pagination
+     */
+    private $pagination;
+
+    /**
+     * ContactUsService constructor.
+     *
+     * @param ContactUsFilter $contactUsFilter
+     * @param ContactUsMapper $contactUsMapper
+     * @param Paginator       $pagination
+     */
+    public function __construct(
+        ContactUsFilter $contactUsFilter,
+        ContactUsMapper $contactUsMapper,
+        Paginator       $pagination)
+    {
+        $this->contactUsFilter = $contactUsFilter;
+        $this->contactUsMapper = $contactUsMapper;
+        $this->pagination      = $pagination;
+    }
+
+    /**
+     * @param int $page
+     * @param int $limit
+     *
+     * @return Paginator
+     */
+    public function getPagination($page = 1, $limit = 10)
+    {
+        $this->pagination->setCurrentPageNumber($page);
+        $this->pagination->setItemCountPerPage($limit);
+
+        return $this->pagination;
+    }
+
+    /**
+     * Return all records from db.
+     *
+     * @return \Zend\Db\ResultSet\ResultSet
+     */
+    public function getAll()
+    {
+        return $this->contactUsMapper->select();
+    }
+
+    /**
+     * @param mixed $contactId
+     *
+     * @return array|\ArrayObject|null
+     */
+    public function getById($contactId)
```

Comment:
Here (and in all code generaly) you can use PHP7 return type.
there will be recommended to return ContactUs entity. 
In that case you should check if result is null and throw exception if it is.
Mapper will return ContactUs by default. 

We are planning to add return types in all code :)